pkg/apis/pravega/v1beta1/pravegacluster_types.go

```diff
@@ -601,12 +601,19 @@ func (p *PravegaCluster) migrateBookkeeper(srcObj *v1alpha1.PravegaCluster) erro
 	}
 
 	log.Printf("Created Bookkeeper CR by name %s", b.Name)
-	err = migrateConfigMap(srcObj, b)
+	configmap := fmt.Sprintf("%s-configmap", srcObj.Name)
+	err = migrateConfigMap(srcObj, b, configmap)
 	if err != nil {
-		log.Fatalf("Error releasing BK CM %v", err)
+		log.Fatalf("Error releasing BK CM %s %v", configmap, err)
 		return err
 	}
-	log.Print("Migrated Bookkeeper ConfigMap.")
+	configmap = nameForBookie(srcObj.Name)
```

Comment:
why are we migrating config map 2 times here

Reply:
They are 2 different configmaps